app/assets/stylesheets/components/_class-card.scss

```diff
@@ -0,0 +1,37 @@
+$class-card-padding-y: $small-spacing;
+$class-card-padding-2y: $base-spacing;
+
+.class-card {
+  @include margin($base-spacing null);
+  border: $thickest-border-width solid $black-haze;
+  padding: $class-card-padding-2y;
```

Comment:
Not sure what the need is for the `y`? At first I thought it was because it could only be applied vertically, but in this case it's also used on the horizontal.